### ISSUE TITLE
internal server error in NER browser when deleting an annotation set

### ISSUE THREAD

**Issue by mr-martian** [annotation/text-fabric#82]
```
Traceback (most recent call last):
  File "/home/daniel/.local/lib/python3.10/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/daniel/.local/lib/python3.10/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/daniel/.local/lib/python3.10/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/daniel/.local/lib/python3.10/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/daniel/text-fabric/tf/browser/ner/web.py", line 79, in serveNerX
    return serveNer(web)
  File "/home/daniel/text-fabric/tf/browser/ner/serve.py", line 324, in serveNer
    serve.actionsFull()
  File "/home/daniel/text-fabric/tf/browser/ner/serve.py", line 74, in actionsFull
    self.setHandling()
  File "/home/daniel/text-fabric/tf/browser/ner/serve.py", line 231, in setHandling
    messages.extend(annotate.setDel(deleteAnnoSet))
  File "/home/daniel/text-fabric/tf/browser/ner/sets.py", line 232, in setDel
    if self.annotSet == delSet:
AttributeError: 'Annotate' object has no attribute 'annotSet'
```

**Comment by dirkroorda** [comment]
A typo. Will be fixed in the next release.

**Comment by dirkroorda** [comment]
Fixed in 12.1.3
